Packs/PcapAnalysis/pack_metadata.json

```diff
@@ -11,7 +11,9 @@
     ],
     "tags": [
         "pcap",
-        "mine"
+        "mine",
```

Comment:
remove "mine" please, you can add it to keywords instead.

Reply:
Done